content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -0,0 +1,1131 @@
+
+```rust
+camera.target = RenderTarget::Image(image_handle);
+```
+
+This enables too many scenarios to count: portals, rendering UI to a texture and rendering it in 3d space, in-game security cameras, real-time player portraits rendered in UI widgets ... the sky is the limit!
+
+Here is an example of a "portal effect":
+
+<video controls loop><source  src="portals.mp4" type="video/mp4"/></video>
+
+This accomplished by rendering a second camera to a texture, synchronizing its orientation with the main player's camera, and using that texture in the main camera's scene.
+
+### Split Screen
+
+Each [`Camera`] now has an optional [`Viewport`], which if set will draw to a section of a [`RenderTarget`] instead of the whole. If you spawn two active cameras and set each camera's [`Viewport`] to draw to half of the window, you have simple, painless split screen!  
+
+![split screen](split_screen.png)
+
+### Layered Rendering
+
+Cameras can now be layered on top of each other using the new "camera priority" field
+
+```rust
+// This camera defaults to priority 0 and is rendered "first" / "at the back" 
+commands.spawn_bundle(Camera3dBundle::default());
+commands.spawn_bundle(Camera3dBundle {
+    camera_3d: Camera3d {
+        // don't clear the color while rendering this camera
+        clear_color: ClearColorConfig::None,
+        ..default()
+    },
+    camera: Camera {
+        // renders after / on top of the main camera
+        priority: 1,
+        ..default()
+    },
+    ..default()
+});
+```
+
+"Priority" determines the order cameras are drawn in. The "portal" effect in the [render to textures example above](#render-to-textures) uses priority to render the "portal" camera first, ensuring it is ready to be used by the main camera.
+
+Here is a simple example of two cameras rendering to the same window:
+
+![camera layering](camera_layering.png)
+
+This can be used for things like "custom UI passes", "minimaps", etc.
+
+### Ergonomic Target Size Access
+
+Working with cameras is now also much easier. They now store their [`RenderTarget`] size locally, which makes retrieving the size much simpler:
+
+```rust
+/// Much nicer than needing to look up the size on the target Window or Image manually,
+/// like you had to in previous Bevy versions!
+let target_size = camera.logical_target_size();
+let viewport_size = camera.logical_viewport_size();
+```
+
+This also means that converting world to screen coordinates for a camera is much easier than it used to be:
+
+```rust
+// Bevy 0.7 (old)
+camera.world_to_screen(windows, images, camera_transform, world_position);
+
+// Bevy 0.8 (new)
+camera.world_to_viewport(camera_transform, world_position);
+```
+
+### New Camera Bundles
+
+The old `OrthographicCameraBundle` and `PerspectiveCameraBundle` have been replaced with [`Camera3dBundle`] and [`Camera2dBundle`]. In most cases migration should be as simple as replacing the old names with the new ones.
```

Comment:
Maybe note that for 3D the projection types are an enum and orthographic is still available?

Reply:
I agree, because right now it looks like old and new cameras are different things.